frontend/src/metabase/embedding/embedding-hub/components/StepperWithCards/StepperWithCards.module.css

```diff
@@ -22,6 +22,9 @@
 .step.stepDone .stepIcon {
   color: var(--mb-color-bg-white);
   background: var(--mb-color-success-darker);
+
+  /** Prevent blue border color from affecting the done icon */
+  border-color: transparent;
```

Comment:
Without this, we get the ugly blue border on the icon.

<img width="1130" height="818" alt="CleanShot 2568-09-18 at 00 23 21@2x" src="https://github.com/user-attachments/assets/eb0f3c6e-f0b0-4b54-a627-a6a603ceea18" />

Reply:
Maybe add this rule within `.stepCard.nextStepCard` to make this more explicit?